geometry/rotation_test.cpp

```diff
@@ -247,5 +253,34 @@ TEST_F(RotationTest, SerializationSuccess) {
   EXPECT_EQ(rotation_a_(vector_), r(vector_));
 }
 
+TEST_F(RotationTest, Basis) {
+  Vector<double, World> a = Normalize(Vector<double, World>({1, 1, -1}));
+  Vector<double, World> b = Normalize(Vector<double, World>({1, 0, 1}));
+  Bivector<double, World> c = Wedge(a, b);
+
+  Rotation<World, World1> const to_world1(a, b, c);
+  EXPECT_THAT(to_world1(a),
+              Componentwise(
+                  AlmostEquals(1, 1),
```

Comment:
This could probably fit on the previous line (with indenting the following parameters).